backend/src/v5/services/eventsListener/components/modelEvents.js

```diff
@@ -38,6 +39,12 @@ const queueTasksCompleted = async ({
 	try {
 		const { _id: projectId } = await findProjectByModelId(teamspace, model, { _id: 1 });
 		await newRevisionProcessed(teamspace, UUIDToString(projectId), model, corId, value, user, containers);
+
+		const { tag, author, timestamp } = await getRevisionByIdOrTag(teamspace, model, corId,
```

Comment:
Not the best place i don't think.

We might be processing a failed model import, in which case you're still attempting to trigger a new revision event here.

I think its' best if `newRevisionProcessed ` trigger a` NEW_REVISION` event upon model processing succeeded, and we catch it to create the chat message, it also knows whether it's a container so you don't have to do `getModelById`